projects/plugins/jetpack/class.jetpack.php

```diff
@@ -924,6 +924,7 @@ public function late_initialization() {
 
 		Partner::init();
 		My_Jetpack_Initializer::init();
+		Jetpack_Backup::initialize();
```

Comment:
```suggestion
				if ( ( new Connection_Manager() )->is_connected() ) {
			Jetpack_Backup::initialize();
		}
```

Could problem be solved by conditionally loading Backup package in Jetpack depending on the connection state? Something along the suggested lines.

Reply:
Something like that, but that will conditionally add the menu or not, and not prevent the whole package from being loaded.

Something here: 

https://github.com/Automattic/jetpack/blob/5bfe3a44588fe51475c6f9b9a807d2a54cc3e406/projects/packages/backup/src/class-jetpack-backup.php#L75

If only Jetpack plugin is active and backup is not enabled, don't add the menu...

Reply:
Looking at it again and the Jetpack_Backup::initialize(); doesn't do much more other than initializing the UI, which is weird, I thought it would initialize other parts of the Backup feature and that's why I suggested to make a more defined condition.

Well, we need to have a closer look